core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -105,6 +105,10 @@
                                   style="margin-left: 10px;">
         </exploration-summary-tile>
       </div>
+
+      <div class="conversation-skin-back-to-collection-container">
```

Comment:
I'm not sure that's necessary -- we have other stuff on this page that only displays in the context of a collection. @mdew192837, can you take a look at `$scope.collectionSummary` and read the JS code in order to figure out in what circumstances it is loaded?